Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Basic docker image for Monocle
+# Usage:
+#   docker build -t Monocle
+#   docker run -d --name Monocle -P Monocle
+
+FROM python:3.6
```

Comment:
I did try to build it with alpine but was running into some missing requirements to install monocle packages so fell back on the full python:3.6